frontend/vue/components/Syllabus/SyllabusView.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <section class="syllabus-view">
+    <h1 class="syllabus-view__title">
+      {{ syllabus.title }}
+    </h1>
+    <h2 class="syllabus-view__section-title">
+      General Information
+    </h2>
+    <SyllabusGeneralInformation class="syllabus-view__general-information">
+      <template #instructor>
+        {{ syllabus.instructor }}
+      </template>
+      <template #location>
+        {{ syllabus.location }}
+      </template>
+      <template #university>
+        {{ syllabus.university }}
+      </template>
+      <template #office-hours>
+        {{ syllabus.officeHours }}
+      </template>
+      <template #class-hours>
+        {{ syllabus.classHours }}
+      </template>
+      <template #email>
+        {{ syllabus.email }}
+      </template>
+    </SyllabusGeneralInformation>
+    <div
+      v-for="(workPackage, idx) in syllabus.workPackages"
```

Comment:
interested in what is a `workPackage` 

(I'm okay w/ it, but would just like to understand what it means since I'm new to the term...)

is this like a course? module?

Reply:
Good question!
I thought of a better name than a course to differentiate between our courses and their chapter selection.
The first approach was `weeks`, but it could be two weeks or a semester or maybe don't have a time constraint.
I decided to call it `workBlocks` or `workPackages` because it is an amount of work that has some coherence and should be worked "in block".
If you have any other suggestions, I will accept them.

Reply:
I'm also leaning towards "module" here. It's also closer to my academic experience.

@Tansito, since ideally we should use the same vocabulary here and in the database, do you think that this name makes sense for this blocks defining the workload of chapters during a specified period?

![image](https://user-images.githubusercontent.com/22047320/158357638-b351306a-9f75-4896-8e56-d50800f986ed.png)